docs: custom selectors registration (#20687)
microsoft/playwright: docs/src/extensibility.md

content scripts. Note that running as a content script is not guaranteed when the engine is used together with other
custom engines.

> Selectors must be registerd before the page that uses it is created, otherwise the

Reviewer: ```suggestion
Selectors must be registered before creating the page.
```

  }
});

// Register selectors before any page is created.
test.beforeAll(async ({ playwright }) => {
  // Register the engine. Selectors will be prefixed with "tag=".
  await playwright.selectors.register('tag', createTagNameEngine);
});

test('selector engine test', async ({ page }) => {
  // Now we can use 'tag=' selectors.
  const button = page.locator('tag=button');
  await button.click();

>   // We can combine it with other selector engines using `>>` combinator.

Reviewer: ```suggestion
  // We can combine it with built-in locators.
```
